### PR TITLE
LPS-154067 [CP 2.0] Fix Translate Keys for Customer Portal Localization

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

**Review by thaynam — APPROVED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/en_US.json [hunk 1]
```diff
@@ -71,27 +80,37 @@
...
 	"generate-x-keys": "Generate {0} Keys",
 	"germany": "Germany",
 	"go-to-product-console": "Go to Product Console",
 	"go-to-workspace": "Go to Workspace",
+	"gold": "Gold",
 	"group-id-confirmation": "Group ID Confirmation",
 	"hamina-finland": "Hamina, Finland",
 	"hide": "Hide",
+	"host-name": "Host Name",
+	"host-name-cluster-size": "Host Name / Cluster Size",
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/en_US.json:113]: 👁️

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Comment by jairmedeiros** [comment]
ci:forward

**Comment by jairmedeiros** [comment]
Just starting reviewing :)

**Comment by jairmedeiros** [comment]
resend https://github.com/pt-liferay-solutions/liferay-portal/pull/1396

**Pull request closed without merging**